tests/enums/test_enum.py

```diff
@@ -67,3 +67,27 @@ def test_raises_error_when_using_enum_with_a_not_enum_class():
         @strawberry.enum
         class NormalClass:
             hello = "world"
+
+
+def test_can_deprecate_enum_values():
```

Comment:
thanks for adding a test! can we also add one for the schema?

we can add a test in https://github.com/strawberry-graphql/strawberry/blob/main/tests/schema/test_enum.py

you can use this query to test for the deprecation:

```graphql
{
  __type(name:"SOMETHING") {
    enumValues {
      isDeprecated
      deprecationReason
    }
  }
}
```

Reply:
Hmm the test I've written fails but actually this passes 👇 
`assert (len(result.data["__type"]["enumValues"]) == 2)  # deprecated value dissapears?!`

Am I missing something?! I will probably try to look at this next weekend with fresh eyes... but if you see something obvious that I'm missing I'm all ears.